src/commands/ExecCommand.js

```diff
@@ -82,6 +82,7 @@ export default class ExecCommand extends Command {
       shell: true,
       env: Object.assign({}, process.env, {
         LERNA_PACKAGE_NAME: pkg.name,
+        LERNA_ROOT_PATH: process.cwd(),
```

Comment:
This should be `this.repository.rootPath`, it isn't always the case that a lerna command is run from the same location as `lerna.json`.